src/Rpc.proto

```diff
@@ -204,7 +204,7 @@ message GetMempoolRequest {
 }
 
 message GetMempoolResponse {
-    map<string, string> info = 1;
+    repeated string mempool = 1;
```

Comment:
Not sure what's going on here? any details

Reply:
I just simplified the data sent.  I found there is no use for the MapField which is basically a key value pair that included the an id (key) and the transaction (value). The list returned is sufficient for the response handler to function properly.

Reply:
Does this mean you're doing the formatting node side rather in the cli?

Reply:
No. I was creating a list of key/value pairs.  Currently, I get the mempool content and return them all as is to the CLI to handle/format in the way the response handler on the CLI side see fit.

Reply:
Maybe @monsieurleberre is best to comment on this if he can.